pkg/deploy/deploy.go

```diff
@@ -43,21 +43,35 @@ func Deploy(workingDir string, fileReader util.FileReader, environmentsFile stri
 	}
 
 	for _, environment := range environments {
-		err := execute(environment, projects, dryRun, workingDir)
-		if err != nil {
-			deploymentErrors[environment.GetId()] = err
+		errors := execute(environment, projects, dryRun, workingDir, continueOnError)
+		if errors != nil {
+			deploymentErrors[environment.GetId()] = errors
 		}
 	}
 
 	for environment, err := range deploymentErrors {
 		if dryRun {
 			util.Log.Error("Validation of %s failed with error %s\n", environment, err)
```

Comment:
I guess we have to change the text here as well. 
Maybe we should put the number of found errors also in the log.